Example/Podfile

```diff
@@ -16,3 +16,18 @@ target 'ChatExample' do
     inherit! :search_paths
   end
 end
+
+# Workaround for https://github.com/pinterest/PINRemoteImage/issues/566
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    next unless %w[PINCache PINRemoteImage].include?(target.name)
+
+    target.build_configurations.each do |config|
```

Comment:
One of the dependencies `PINRemoteImage` (and its subspec PINCache) is not compatible with Xcode 12.
The compiler complains that there's incompatible block pointers.

So I added workaround to disable this warning by passing extra flags.